make_mod.sh

```diff
@@ -3,7 +3,9 @@
 set -o errexit
 trap cleanup EXIT
 
-CURRENT_DIR=$(pwd)
+PROJECT_ROOT="$(dirname -- "$(readlink -f "${BASH_SOURCE[0]}")")"
```

Comment:
This is a more accurate way to get the project's directory than `pwd`, as it will return the correct directory no matter where the script is called from.